drivers/sensor/qdec_nrfx/Kconfig

```diff
@@ -3,7 +3,7 @@
 
 config QDEC_NRFX
 	bool "Nordic QDEC nrfx driver"
-	depends on HAS_HW_NRF_QDEC
+	depends on HAS_HW_NRF_QDEC || HAS_HW_NRF_QDEC0 || HAS_HW_NRF_QDEC1
```

Comment:
The nrfx_qdec driver, and consequently the qdec_nrfx shim for it, currently supports only single QDEC instance (for nRF53 it is `NRF_QDEC0`, which is mapped to `NRF_QDEC` in [nrfx_config_nrf5340_application.h](https://github.com/zephyrproject-rtos/zephyr/blob/master/modules/hal_nordic/nrfx/nrfx_config_nrf5340_application.h#L101)), so I think `HAS_HW_NRF_QDEC1` should not be added here yet, to not bring confusion in this regard.